lib/http.js

```diff
@@ -11,17 +11,23 @@
  * for the `expect` and `should` interfaces.
  */
 
-module.exports = function (chai, _) {
+/*!
+ * Module dependencies.
+ */
+import net from "net"
```

Comment:
the formatting should really be consistent (i.e. missing semi here) but i wonder if we should just do a follow up prettier PR if we don't already have it

Reply:
yeah, setting up prettier would help a lot

We can create these follow-up PRs and then create the v5 release if this is finished (I didn't want to create all of this in this PR, as it would be too much I think)